installer/schema.go

```diff
@@ -23,11 +24,21 @@ type Cluster interface {
 }
 
 type Credential struct {
-	ID        string     `json:"id" ql:"index xID"`
-	Secret    string     `json:"secret"`
-	Name      string     `json:"name"`
-	Type      string     `json:"type"` // enum(aws, digital_ocean)
-	DeletedAt *time.Time `json:"deleted_at,omitempty"`
+	ID         string             `json:"id" ql:"index xID"`
+	Secret     string             `json:"secret"`
+	Name       string             `json:"name"`
+	Type       string             `json:"type"` // enum(aws, digital_ocean, azure)
+	OAuthCreds []*OAuthCredential `json:"oauth_creds,omitempty" ql:"-"`
+	DeletedAt  *time.Time         `json:"deleted_at,omitempty"`
+}
+
+type OAuthCredential struct {
+	ClientID     string     `json:"client_id"` // ClientID <-> Credential.ID
```

Comment:
Credential.ID and OAuthCredential.ClientID are the same, and `==` would probably be better